test/extensions/filters/listener/http_inspector/http_inspector_test.cc

```diff
@@ -40,7 +41,6 @@ class HttpInspectorTest : public testing::TestWithParam<Http1ParserImpl> {
         io_handle_(
             Network::SocketInterfaceImpl::makePlatformSpecificSocket(42, false, absl::nullopt, {})),
         parser_impl_(GetParam()) {}
-  ~HttpInspectorTest() override { io_handle_->close(); }
```

Comment:
Why we remove this line?

Reply:
It was resulting in double close crashes because we can now close when exceeding the buffer.